api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -187,10 +187,10 @@ public DbWorkspace get(String ns, String firecloudName) {
 
   @Override
   public List<WorkspaceResponse> getWorkspacesAndPublicWorkspaces() {
-    Map<String, FirecloudWorkspaceResponse> fcWorkspaces =
-        getFirecloudWorkspaces(ImmutableList.of("accessLevel", "workspace.workspaceId"));
+    Map<String, FirecloudWorkspaceResponse> fcWorkspaces = getFirecloudWorkspaces();
```

Comment:
nit: I'd name this like  `uuidToFirecloudWorkspace`. I'd abbreviate  workspace before  firecloud because of local context.

Reply:
Existing issue, but sure